web/src/app/documentation/sections/Webhooks.md

```diff
@@ -0,0 +1,67 @@
+# Using Webhooks
+
+Webhooks are POST requests to specified endpoints with a content type of `application/json`. Below are example payloads:
+
+### Verification Message
+
+```
+{
+    "AlertType": "Verification",
+    "Summary": "Verification Message",
+    "Details": "This is a verification message from GoAlert",
+    "Code": "283917"
+}
+```
+
+### Test Message (from profile page)
```

Comment:
We should add short descriptions of the message types along with why/when you get them.
```suggestion
### Test Message

Triggered on the profile page by clicking "Send Test"
```